ImageFeed/Auth/AuthViewController.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
+
+protocol AuthViewControllerDelegate: AnyObject{
+    func authViewController(_ vc: AuthViewController, didAuthenticateWithCode code: String)
+}
+
+final class AuthViewController: UIViewController{
+    let WebViewSegueIdentifier = "ShowWebView"
+    
+    weak var delegate: AuthViewControllerDelegate?
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == WebViewSegueIdentifier{
+            guard let webViewViewController = segue.destination as? WebViewViewController else{ fatalError("Failed to prepare for \(WebViewSegueIdentifier)")
+            }
```

Comment:
Если guard не влезает в 120 символов в одной строке, то разбиваем его вот так:
```suggestion
            guard
                let webViewViewController = segue.destination as? WebViewViewController 
            else{ 
               fatalError("Failed to prepare for \(WebViewSegueIdentifier)")
            }
```